servers/xr/xr_interface_extension.cpp

```diff
@@ -316,10 +319,14 @@ void XRInterfaceExtension::end_frame() {
 RID XRInterfaceExtension::get_render_target_texture(RID p_render_target) {
 	// In due time this will need to be enhance to return the correct INTERNAL RID for the chosen rendering engine.
 	// So once a GLES driver is implemented we'll return that and the implemented plugin needs to handle this correctly too.
```

Comment:
FYI @BastiaanOlij @dsnopek - the OpenGL driver is implemented ;)

Reply:
I'm not sure why we have this. Is this a utility function meant to be used by child classes of `XRInterfaceExtension`? If so, I don't think they need it, there's `RenderingServer::texture_get_rd_texture()` exposed. Maybe this is something that can be deprecated?

Given that we have OpenXR in Godot itself now, it's possible `XRInterfaceExtension` was never fully baked...

Reply:
Or, hm, this takes the render target, not the RID of the `RenderingServer` texture. So, maybe we do need this as a utility function? Would be nice for it to be on `RenderingServer`, though, and not here

Reply:
This is something I wrote waaaaaaaaaaay at the beginning and probably came from translating the implementation in Godot 3 to Godot 4. I'm not even sure if there are implementations that use this. Maybe the TiltFive GDExtension? Though that works with OpenGL so more reason to think this is a defunct method...